docs/index.rst

```diff
@@ -2,42 +2,74 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
-data analysis environment for Python.
+``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
```

Comment:
Do we want to enforce all sunpy library references are marked up like this?

Reply:
I think it looks good here - if we want to disucss enforcing (or not) as a rule, worth opening a new issue?

Reply:
I had a look over the docs and its a bit all over the place how sunpy is stylised: ``sunpy``, SunPy, sunpy core (package), the sunpy project etc.

If there are rules to follow or rules to be made then we should do that. I don't care how it is stylised here really.

Reply:
We need to enforce this down the road but for now, we don't need to block.